setup.py

```diff
@@ -1,21 +1,21 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from pip.download import PipSession
-from pip.req import parse_requirements
-
 from setuptools import find_packages, setup
 
 with open('README.rst', 'r') as f:
     long_description = f.read()
 
 
+def parse_requirements(filename):
+    lineiter = [line.strip() for line in open(filename)]
+    return [line for line in lineiter if line and not line.startswith("#")]
+
+
 def get_requirements():
     rtd_file = open('docs/rtd-requirements.txt', 'w')
```

Comment:
Why not make this a with block as well to avoid having to call close explicitly?

Reply:
Good idea.  Looking at the code again, a compound with statement does make more sense.